src/common/fetch.js

```diff
@@ -73,6 +72,23 @@ export async function fetchPost(path, body, otherOptions = {}, headers = {}) {
 
 	return data;
 }
+export async function fetchPostFormData(path, body, otherOptions = {}, headers = {}) {
+
+	const url = `${API_URL}/${path}`;
+	const options = {
+		method: "POST",
+		headers: {
+			...bearer,
+			...headers,
+		},
+		body: body,
```

Comment:
저번에두 이렇게 쓰셨길래 하나 알려드리자면.. 객체에서 key, value가 같으면 한 번만 써도 돼용!
아래 예시
```suggestion
		body,
```

Reply:
수정했습니다.